python/sglang/srt/managers/tokenizer_manager.py

```
@@ -895,6 +895,9 @@ async def update_weights_from_disk(
             obj.load_format = self.server_args.load_format
         logger.info("Start update_weights. Load format=%s", obj.load_format)
 
+        if obj.abort_all_requests:
+            self.abort_request(rid="", abort_all=True)
```

if we want to abort all requests, we can just call `self.abort_request(abort_all=True)` instead of passing a empty `rid` into the function

Hence I would suggest making `rid` a optional input.

---

> if we want to abort all requests, we can just call `self.abort_request(abort_all=True)` instead of passing a empty `rid` into the function
> 
> Hence I would suggest making `rid` a optional input.

Got it, I've already changed that.